CHANGELOG.unreleased.md

```diff
@@ -17,6 +17,7 @@
 - [Plugin/auth-saml] Certificate input support multiline (PR [#6403](https://github.com/vatesfr/xen-orchestra/pull/6403))
 - [Backup] Launch Health Check after a full backup (PR [#6401](https://github.com/vatesfr/xen-orchestra/pull/6401))
 - [Backup] Fix `Lock file is already being held` error when deleting a VM backup while the VM is currently being backed up
+- [Backup] Fix Backup stoping on `remote is disabled` error when other remotes are working [#6347](https://github.com/vatesfr/xen-orchestra/issues/6374) (PR [#6430](https://github.com/vatesfr/xen-orchestra/pull/6430))
```

Comment:
```suggestion
- [Backup] Ignore disabled remotes instead of failing the execution [#6347](https://github.com/vatesfr/xen-orchestra/issues/6374) (PR [#6430](https://github.com/vatesfr/xen-orchestra/pull/6430))
```